Rework wishlist into a proper table (#1395)
YunoHost/doc: apps_wishlist.md

| qBittorrent |  | [Upstream](https://github.com/qbittorrent/qBittorrent) |  |
| [Questions2answer](https://www.question2answer.org/) |  |  |  |
| [racktables](https://racktables.org) |  | [Upstream](https://github.com/RackTables/racktables) |  |
| Radarr |  | [Upstream](https://github.com/Radarr/Radarr) |  |
| [redmine](https://www.redmine.org/) | A flexible project management web application |  | [Package Draft](https://github.com/YunoHost-Apps/redmine_ynh) |
| RemoteStorage | A remoteStorage server implementation written in PHP | [Upstream](https://github.com/fkooman/php-remote-storage) | [Package Draft](https://github.com/YunoHost-Apps/RemoteStorage_ynh) |
| [Request Tracker](https://bestpractical.com) |  | [Upstream](https://github.com/bestpractical/rt) |  |
| [Restya](https://restya.com) |  | [Upstream](https://github.com/RestyaPlatform/board/) |  |
| [Retroshare](https://retroshare.cc/) |  | [Upstream](https://github.com/RetroShare/RetroShare) |  |
| [roadiz](https://www.roadiz.io) | Create a modern Website |  | [Package Draft](https://github.com/YunoHost-Apps/roadiz_ynh) |

> | [rocketchat](https://rocket.chat) | the ultimate chat platform |  | [Package Draft](https://github.com/YunoHost-Apps/rocketchat_ynh) |

Reviewer: I guess we can also close this request #1291

| headphones | Automatic music downloader |  | [Package Draft](https://github.com/YunoHost-Apps/headphones_ynh) |
| Helpy |  | [Upstream](https://github.com/helpyio/helpy) |  |
| [Hexo](https://hexo.io/) |  | [Upstream](https://github.com/hexojs/hexo) |  |
| Hometown |  | [Upstream](https://github.com/hometown-fork/hometown) |  |
| [htpc-manager](https://htpc.io) | Manage your HTPC from anywhere |  | [Package Draft](https://github.com/scith/htpc-manager_ynh) |
| huginn | Build agents that monitor and act on your behalf |  | [Package Draft](https://github.com/YunoHost-Apps/huginn_ynh) |
| [humhub](https://www.humhub.org) | Enterprise Social Network |  | [Package Draft](https://github.com/yunohost-apps/humhub_ynh) |
| [Icecast 2](https://www.icecast.org) |  | [Upstream](https://gitlab.xiph.org/xiph/icecast-server/) |  |
| [ikiwiki](https://ikiwiki.info) |  |  |  |
| internetarchive | Offline version of the Internet Archive | [Upstream](https://github.com/internetarchive/dweb-mirror) | [Package Draft](https://github.com/mitra42/internetarchive_ynh) |
| Invidious |  | [Upstream](https://github.com/omarroth/invidious) |  |

> | [Invoice Ninja](https://www.invoiceninja.com) |  | [Upstream](https://github.com/invoiceninja/invoiceninja) |  |

Reviewer: ```suggestion

```
We can remove InvoiceNinja out of the list as it is up and running https://github.com/YunoHost-Apps/invoiceninja_ynh

| microblog.pub |  | [Upstream](https://github.com/tsileo/microblog.pub) |  |
| [mindmaps](https://www.mindmaps.app/) | Tool for making mind maps |  | [Package Draft](https://github.com/YunoHost-Apps/mindmaps_ynh) |
| [mineweb](https://mineweb.org/) |  |  | [Package Draft](https://github.com/YunoHost-Apps/mineweb_ynh) |
| miniflux | Minimal RSS reader |  | [Package Draft](https://github.com/mat-mo/miniflux_ynh) |
| [Mirakel](https://mirakel.azapps.de/taskwarrior.html) |  | [Upstream](https://github.com/GothenburgBitFactory/taskwarrior) |  |
| modernpaste | A modern, feature-rich Pastebin alternative | [Upstream](https://github.com/LINKIWI/modern-paste) | [Package Draft](https://github.com/YunoHost-Apps/modernpaste_ynh) |
| [Modoboa](https://modoboa.org) |  | [Upstream](https://github.com/modoboa/) |  |
| [mopidy](https://www.mopidy.com/) | An extensible music server |  | [Package Draft](https://github.com/YunoHost-Apps/mopidy_ynh) |
| [mosquitto](https://mosquitto.org) |  |  | [Package Draft](https://github.com/YunoHost-Apps/mosquitto_ynh) |
| MotionEye |  | [Upstream](https://github.com/ccrisan/motioneye) |  |

> | mumble_admin_plugin | Mumble-server web interface |  | [Package Draft](https://github.com/matlink/mumble_admin_plugin_ynh) |

Reviewer: ```suggestion
| mumble_admin_plugin | Mumble-server web interface |  | [Package Draft](https://github.com/matlink/mumble_admin_plugin_ynh) |
| Mumble-web  |  Mumble web interface |  [Upstream](https://github.com/Johni0702/mumble-web) | [Package Draft](https://github.com/YunoHost-Apps/mumbleserver_ynh/pull/32) | 
```
Add Mumble Web from request #1315

# Apps wishlist

The following list is a compiled wishlist of applications that would be nice-to-have.

You can [contribute to this list by adding something you'd like to be packaged](/write_documentation), or learn more on [how to package applications](/packaging_apps) to create a new package or improve a package draft.

> <div class="alert alert-info">Before adding an app in this wishlist, please check your app is not <a href="/apps">already available in Yunohost's app catalog</a></div>

Reviewer 2: ```suggestion
<div class="alert alert-info">Before adding an app in this wish-list, please check your app is not <a href="/apps">already available in YunoHost's app catalogue</a></div>
```

| Name   | Description | Upstream | Package draft |
| ------ | ----------- | -------- | ------------- |
| 10er10 |  | [Upstream](https://github.com/dready92/10er10) |  |

> | [abantecart](https://abantecart.com) | Create a E-commerce Website |  | [Package Draft](https://github.com/YunoHost-Apps/abantecart_ynh) |

Reviewer 2: ```suggestion
| [abantecart](https://abantecart.com) | Create an e-commerce website |  | [Package Draft](https://github.com/YunoHost-Apps/abantecart_ynh) |
```

| adguard |  | [Upstream](https://github.com/AdguardTeam/AdGuardHome) | [Package Draft](https://github.com/YunoHost-Apps/adguard_ynh) |

> | adhocserver | An adhoc Server for the sony PSP |  | [Package Draft](https://github.com/matlink/adhocserver_ynh) |

Reviewer 2: ```suggestion
| adhocserver | An ad hoc server for the Sony PSP |  | [Package Draft](https://github.com/matlink/adhocserver_ynh) |
```
Not 100% sure this isn't the "adhoc party", but as per https://manuals.playstation.net/document/en/psp/current/settings/adhoc.html

| [agora](https://www.agora-project.net/) | Create collaborative and creative digital space |  | [Package Draft](https://github.com/YunoHost-Apps/agora_ynh) |
| [Ajenti](https://ajenti.org) |  | [Upstream](https://github.com/ajenti/ajenti/) |  |
| Akaunting | Manage payments/invoices/expenses | [Upstream](https://github.com/akaunting/akaunting) |  |

> | [askbot](https://askbot.com/) | Questions & Answers |  | [Package Draft](https://github.com/zamentur/askbot_ynh) |

Reviewer 2: ```suggestion
| [askbot](https://askbot.com/) | Questions and answers |  | [Package Draft](https://github.com/zamentur/askbot_ynh) |
```

| chtickynotes | Note manager |  | [Package Draft](https://github.com/YunoHost-Apps/chtickynotes_ynh) |
| [Citadel-suite](https://www.citadel.org) | Groupware platform |  |  |
| [Cockpit](https://cockpit-project.org/) |  |  | [Package Draft](https://github.com/YunoHost-Apps/cockpit_ynh) |
| coin | Member dashboard for non profit ISP | [Upstream](https://code.ffdn.org/FFDN/coin/) | [Package Draft](https://github.com/YunoHost-Apps/coin_ynh) |
| Commafeed |  | [Upstream](https://github.com/Athou/commafeed) |  |
| [Converse.js](https://conversejs.org) | Web client for XMPP | [Upstream](https://github.com/jcbrand/converse.js) |  |
| cops | Calibre OPDS |  | [Package Draft](https://github.com/YunoHost-Apps/cops_ynh) |
| coquelicot | A “one-click” file sharing web application |  | [Package Draft](https://github.com/YunoHost-Apps/coquelicot_ynh) |
| [Couchdb](https://couchdb.apache.org/) | Database management |  |  |
| [Cozy](https://cozy.io/en/) |  |  |  |
| Croodle | Vote for a schedule / polling | [Upstream](https://github.com/jelhan/croodle) |  |
| Darkwire.io |  | [Upstream](https://github.com/seripap/darkwire.io) |  |

> | [democracyos](https://democracyos.org/) | Vote / take decision in a collective |  | [Package Draft](https://github.com/YunoHost-Apps/democracyos_ynh) |

Reviewer 2: ```suggestion
| [democracyos](https://democracyos.org/) | Vote / make decisions in a collective |  | [Package Draft](https://github.com/YunoHost-Apps/democracyos_ynh) |
```

| [diasporadocker](https://diasporafoundation.org/) | A open and powerfull social network |  | [Package Draft](https://github.com/aymhce/diasporadocker_ynh) |
| DirectoryLister |  | [Upstream](https://github.com/DirectoryLister/DirectoryLister) |  |
| django_app | Skeleton for django apps |  | [Package Draft](https://github.com/Jojo144/django_app_ynh) |
| docker_container | Deploy applications with Docker |  | [Package Draft](https://github.com/scith/docker_container_ynh) |
| docker_rstudio | A web development environment for statistics with R |  | [Package Draft](https://github.com/scith/docker_rstudio_ynh) |
| docker-registry |  | [Upstream](https://github.com/docker/distribution/) | [Package Draft](https://github.com/plopoyop/docker-registry_ynh) |
| dockerui | Deploy applications with Docker |  | [Package Draft](https://github.com/YunoHost-Apps/dockerui_ynh) |
| [domoticz](https://domoticz.com) | Home automation system web application |  | [Package Draft](https://github.com/anubister/domoticz_ynh) |
| Draw.io | Online diagram creation | [Upstream](https://github.com/jgraph/drawio) |  |

> | [duniter](https://duniter.org) | Participate to the duniter blockchain |  | [Package Draft](https://github.com/YunoHost-Apps/duniter_ynh) |

Reviewer 2: ```suggestion
| [duniter](https://duniter.org) | Participate to the Duniter blockchain |  | [Package Draft](https://github.com/YunoHost-Apps/duniter_ynh) |
```
As per https://duniter.org/en/introduction/

> | [duniter](https://duniter.org) | Participate to the duniter blockchain |  | [Package Draft](https://github.com/YunoHost-Apps/duniter_ynh) |

Reviewer 2: ```suggestion
| [duniter](https://duniter.org) | Participate in the Duniter blockchain |  | [Package Draft](https://github.com/YunoHost-Apps/duniter_ynh) |
```
As per https://duniter.org/en/introduction/

> | dynette | Dynette is a dyndns server | [Upstream](https://github.com/YunoHost/dynette) | [Package Draft](https://github.com/YunoHost-Apps/dynette_ynh) |

Reviewer 2: ```suggestion
| dynette | Dynette is a DDNS server | [Upstream](https://github.com/YunoHost/dynette) | [Package Draft](https://github.com/YunoHost-Apps/dynette_ynh) |
```
As per https://en.wikipedia.org/wiki/Dynamic_DNS

| [FEX](https://fex.rus.uni-stuttgart.de/) |  |  |  |
| [Filestash](https://www.filestash.app/) |  | [Upstream](https://github.com/mickael-kerjean/filestash) |  |
| FitTrackee |  | [Upstream](https://github.com/SamR1/FitTrackee) |  |
| flask | Skeleton for flask apps |  | [Package Draft](https://github.com/YunoHost-Apps/flask_ynh) |
| [foodsoft](https://foodcoops.github.io/foodsoft-hosting/) | Manage a non-profit food cooperative |  | [Package Draft](https://github.com/YunoHost-Apps/foodsoft_ynh) |
| [Fossil](https://www.fossil-scm.org) |  |  |  |
| framaestro | Online service aggregator | [Upstream](https://framagit.org/framasoft/framaestro) | [Package Draft](https://github.com/YunoHost-Apps/framaestro_ynh) |
| framaestro_hub | Online service aggregator hub | [Upstream](https://github.com/mozilla/togetherjs.git) | [Package Draft](https://github.com/YunoHost-Apps/framaestro_hub_ynh) |
| [freeboard](https://freeboard.io/) | A real-time dashboard builder for IOT |  | [Package Draft](https://github.com/YunoHost-Apps/freeboard_ynh) |
| [freepbx](https://freepbx.org) | VOIP |  | [Package Draft](https://github.com/YunoHost-Apps/freepbx_ynh) |

> | [ftssolr](https://wiki.dovecot.org/Plugins/FTS/Solr) | Full Text Search via IMAP through a Solr indexing server |  | [Package Draft](https://github.com/YunoHost-Apps/ftssolr_ynh) |

Reviewer 2: ```suggestion
| [ftssolr](https://wiki.dovecot.org/Plugins/FTS/Solr) | Full-text search via IMAP through a Solr indexing server |  | [Package Draft](https://github.com/YunoHost-Apps/ftssolr_ynh) |
```

> | galette | Membership management for non profit organizations |  | [Package Draft](https://github.com/YunoHost-Apps/galette_ynh) |

Reviewer 2: ```suggestion
| galette | Membership management for non-profit organizations |  | [Package Draft](https://github.com/YunoHost-Apps/galette_ynh) |
```

| [Gancio](https://gancio.org/) |  | [Upstream](https://framagit.org/les/gancio) |  |
| gateone | HTML5-powered terminal emulator and SSH client |  | [Package Draft](https://github.com/Kloadut/gateone_ynh) |
| [Geneweb](https://geneweb.tuxfamily.org) |  | [Upstream](https://github.com/geneweb/geneweb) |  |
| GetSimple CMS |  | [Upstream](https://github.com/GetSimpleCMS/GetSimpleCMS) |  |
| gitolite | Git server |  | [Package Draft](https://github.com/matlink/gitolite_ynh) |
| gitrepositories |  |  | [Package Draft](https://github.com/YunoHost-Apps/gitrepositories_ynh) |
| gitweb | Git forge (c.f. Gitolite) |  | [Package Draft](https://github.com/matlink/gitweb_ynh) |
| [glitch-soc](https://glitch-soc.github.io/docs/) | A libre and federated social network |  | [Package Draft](https://github.com/Tagadda/glitch-soc_ynh) |
| glpi | Create a beautiful blog or website easily |  | [Package Draft](https://github.com/abeudin/glpi_ynh) |
| gnusocial | Create a federated comunication node |  | [Package Draft](https://github.com/YunoHost-Apps/gnusocial_ynh) |
| [Goaccess](https://goaccess.io) | Web log analyzer | [Upstream](https://github.com/allinurl/goaccess) |  |

> | gogs_webhost | Turn a gogs repository to static web hosting |  | [Package Draft](https://github.com/YunoHost-Apps/gogs_webhost_ynh) |

Reviewer 2: ```suggestion
| gogs_webhost | Turn a Gogs repository to static web hosting |  | [Package Draft](https://github.com/YunoHost-Apps/gogs_webhost_ynh) |
```
As per https://manuals.playstation.net/document/en/psp/current/settings/adhoc.html

| [InvoicePlane](https://invoiceplane.com) |  | [Upstream](https://github.com/InvoicePlane/InvoicePlane) |  |
| [IPFS](https://ipfs.io) |  | [Upstream](https://github.com/ipfs/ipfs) |  |
| [Isso](https://posativ.org/isso) | Comment server | [Upstream](https://github.com/posativ/isso) |  |
| jappix_mini | Jappix Mini XMPP chat website integration |  | [Package Draft](https://github.com/YunoHost-Apps/jappix_mini_ynh) |
| [jenkins](https://jenkins.io/index.html) | Extendable continuous integration server |  | [Package Draft](https://github.com/YunoHost-Apps/jenkins_ynh) |
| [Jitsi](https://jitsi.org/) | Video conferencing web application |  | [Package Draft](https://github.com/YunoHost-Apps/jitsi_ynh) |
| [joomla](https://www.joomla.org) | A content management system |  | [Package Draft](https://github.com/YunoHost-Apps/joomla_ynh) |
| [Joplin](https://joplin.cozic.net/) |  | [Upstream](https://github.com/laurent22/joplin) |  |
| Joplin-web |  | [Upstream](https://github.com/foxmask/joplin-web) |  |
| [JS Bin](https://jsbin.com/) |  | [Upstream](https://github.com/jsbin/jsbin) |  |
| kiwiirc | Web IRC client |  | [Package Draft](https://github.com/YunoHost-Apps/kiwiirc_ynh) |

> | [Kiwix](https://www.kiwix.org/) | Offline wikipedia | [Upstream](https://github.com/kiwix/) |  |

Reviewer 2: ```suggestion
| [Kiwix](https://www.kiwix.org/) | Offline Wikipedia or other sites | [Upstream](https://github.com/kiwix/) |  |
```
As per https://www.kiwix.org/en/

| [Known](https://withknown.com) |  | [Upstream](https://github.com/idno/known) |  |
| [kodi](https://kodi.tv) | Media center application |  | [Package Draft](https://github.com/YunoHost-Apps/kodi_ynh) |
| [Koel](https://koel.phanan.net) |  | [Upstream](https://github.com/phanan/koel) |  |
| Kontalk |  | [Upstream](https://github.com/kontalk) |  |
| [Koozip](https://koozic.net) |  | [Upstream](https://github.com/DocMarty84/koozic) |  |
| L'atelier |  |  [Upstream](https://github.com/jbl2024/latelier) |  |
| [LBCAlerte](https://alerte.ilatumi.org/) |  |  | [Package Draft](https://github.com/YunoHost-Apps/LBCAlerte_ynh) |
| lektor | A static website generator |  | [Package Draft](https://github.com/Yunohost-Apps/lektor_ynh) |

> | [Lemmy](https://dev.lemmy.ml) | A federated alternative to reddit | [Upstream](https://github.com/dessalines/lemmy) |  |

Reviewer 2: ```suggestion
| [Lemmy](https://dev.lemmy.ml) | A federated alternative to Reddit | [Upstream](https://github.com/dessalines/lemmy) |  |
```

| [Lessy](https://lessy.io) |  | [Upstream](https://github.com/lessy-community/lessy) |  |
| [LibreOffice Online](https://www.libreoffice.org/download/libreoffice-online/) |  |  |  |
| linuxdash | Low-overhead monitoring web dashboard | [Upstream](https://github.com/afaqurk/linux-dash) | [Package Draft](https://github.com/YunoHost-Apps/linuxdash_ynh) |
| [LiquidSoap](https://www.liquidsoap.info/) | Audio and video streaming language | [Upstream](https://github.com/savonet/liquidsoap) |  |
| LocomotiveCMS |  | [Upstream](https://github.com/locomotivecms/engine) |  |
| [Loomio](https://www.loomio.org) |  | [Upstream](https://github.com/loomio/loomio/) |  |
| lychee | Self-hosted photo-management done right |  | [Package Draft](https://github.com/titoko/lychee_ynh) |
| [MaidSafe](https://maidsafe.net) |  | [Upstream](https://github.com/maidsafe) |  |

> | [mailman3](https://docs.mailman3.org/) | Electronic mailing lists manager |  | [Package Draft](https://github.com/YunoHost-Apps/mailman3_ynh) |

Reviewer 2: ```suggestion
| [mailman3](https://docs.mailman3.org/) | Electronic mailing list manager |  | [Package Draft](https://github.com/YunoHost-Apps/mailman3_ynh) |
```
As per the URL

| [Mailtrain](https://mailtrain.org/) |  | [Upstream](https://github.com/Mailtrain-org/mailtrain) |  |
| [Mautic](https://github.com/mautic/mautic) |  |  |  |
| mediadrop | Video Platform |  | [Package Draft](https://github.com/YunoHost-Apps/mediadrop_ynh) |
| [medusa](https://pymedusa.com/) | Automatic TV shows downloader |  | [Package Draft](https://github.com/guigot/medusa_ynh) |
| microblog.pub |  | [Upstream](https://github.com/tsileo/microblog.pub) |  |
| [mindmaps](https://www.mindmaps.app/) | Tool for making mind maps |  | [Package Draft](https://github.com/YunoHost-Apps/mindmaps_ynh) |
| [mineweb](https://mineweb.org/) |  |  | [Package Draft](https://github.com/YunoHost-Apps/mineweb_ynh) |
| miniflux | Minimal RSS reader |  | [Package Draft](https://github.com/mat-mo/miniflux_ynh) |
| [Mirakel](https://mirakel.azapps.de/taskwarrior.html) |  | [Upstream](https://github.com/GothenburgBitFactory/taskwarrior) |  |
| modernpaste | A modern, feature-rich Pastebin alternative | [Upstream](https://github.com/LINKIWI/modern-paste) | [Package Draft](https://github.com/YunoHost-Apps/modernpaste_ynh) |
| [Modoboa](https://modoboa.org) |  | [Upstream](https://github.com/modoboa/) |  |

> | [mopidy](https://www.mopidy.com/) | An extensible music server |  | [Package Draft](https://github.com/YunoHost-Apps/mopidy_ynh) |

Reviewer 2: Is "extensible" correct?

| [mosquitto](https://mosquitto.org) |  |  | [Package Draft](https://github.com/YunoHost-Apps/mosquitto_ynh) |
| MotionEye |  | [Upstream](https://github.com/ccrisan/motioneye) |  |
| mumble_admin_plugin | Mumble-server web interface |  | [Package Draft](https://github.com/matlink/mumble_admin_plugin_ynh) |
| Mumble-web  |  Mumble web interface |  [Upstream](https://github.com/Johni0702/mumble-web) | [Package Draft](https://github.com/YunoHost-Apps/mumbleserver_ynh/pull/32) | 
| munin | Resource monitoring tool |  | [Package Draft](https://github.com/YunoHost-Apps/munin_ynh) |
| Museek+ |  | [Upstream](https://github.com/eLvErDe/museek-plus) |  |
| [MyBB](https://mybb.com/) |  | [Upstream](https://github.com/mybb) |  |
| mycryptochat | Encrypted IM |  | [Package Draft](https://github.com/mrtino/mycryptochat_ynh) |
| [N8n.io](https://n8n.io) |  |  |  |
| Netrunner |  | [Upstream](https://github.com/mtgred/netrunner) |  |

> | nexusoss | sonatype nexus repository manager OSS |  | [Package Draft](https://github.com/YunoHost-Apps/nexusoss_ynh) |

Reviewer 2: ```suggestion
| nexusoss | Sonatype Nexus Repository manager OSS |  | [Package Draft](https://github.com/YunoHost-Apps/nexusoss_ynh) |
```
As per https://www.sonatype.com/nexus/repository-pro

| [OpenCart](https://www.opencart.com) |  | [Upstream](https://github.com/opencart/opencart) |  |
| [OpenEats](https://open-eats.github.io/) |  | [Upstream](https://github.com/open-eats/OpenEats) |  |
| [openHAB](https://www.openhab.org/) | Smart home platform | [Upstream](https://github.com/openhab) |  |
| openid-simplesamlphp | OpenID provider based on SimpleSAMLphp |  | [Package Draft](https://github.com/julienmalik/openid-simplesamlphp_ynh) |
| [OpenNote](https://foxusa.github.io/OpenNote/OpenNote/) | An alternative to Microsoft OneNote and EverNote |  | [Package Draft](https://github.com/YunoHost-Apps/OpenNote_ynh) |
| openproject |  | [Upstream](https://github.com/opf/openproject) | [Package Draft](https://github.com/moutonjr/openproject_ynh) |
| OpenSourceBilling |  | [Upstream](https://github.com/vteams/open-source-billing) |  |
| [opentracker](https://erdgeist.org/arts/software/opentracker/) |  |  | [Package Draft](https://github.com/YunoHost-Apps/opentracker_ynh) |
| [osada](https://zotlabs.com/osada/) | A decentralized publication platform and social network |  | [Package Draft](https://github.com/YunoHost-Apps/osada_ynh) |

> | [osjs](https://www.os-js.org/) | Desktop you have accesss through your browser |  | [Package Draft](https://github.com/YunoHost-Apps/osjs_ynh) |

Reviewer 2: ```suggestion
| [osjs](https://www.os-js.org/) | Desktop you have access to through your web-browser |  | [Package Draft](https://github.com/YunoHost-Apps/osjs_ynh) |
```

| [osmw](https://www.openstreetmap.org/) | Cartography software |  | [Package Draft](https://github.com/YunoHost-Apps/osmw_ynh) |
| OSRM |  | [Upstream](https://github.com/Project-OSRM/osrm-backend) |  |
| [OX Open-Xchange](https://www.open-xchange.com) | Linux groupware solution | [Upstream](https://github.com/open-xchange/) |  |

> | [pagure](https://pagure.io/pagure) | A git forge |  | [Package Draft](https://github.com/YunoHost-Apps/pagure_ynh) |

Reviewer 2: ```suggestion
| [pagure](https://pagure.io/pagure) | A Git forge |  | [Package Draft](https://github.com/YunoHost-Apps/pagure_ynh) |
```

| Paperless |  | [Upstream](https://github.com/danielquinn/paperless) |  |
| [Paperwork](https://paperwork.rocks) |  |  |  |

> | [Passbolt](https://www.passbolt.com) | Passwords manager | [Upstream](https://github.com/passbolt) |  |

Reviewer 2: ```suggestion
| [Passbolt](https://www.passbolt.com) | Password manager | [Upstream](https://github.com/passbolt) |  |
```

| [Photoprism](https://photoprism.org/) |  | [Upstream](https://github.com/photoprism/photoprism) |  |
| [phpbb](https://www.phpbb.com/) | Bulletin board software | [Upstream](https://github.com/phpbb) | [Package Draft](https://github.com/YunoHost-apps/phpbb_ynh) |
| [phpboost](https://www.phpboost.com/) |  |  | [Package Draft](https://github.com/YunoHost-Apps/phpboost_ynh) |
| [phpipam](https://phpipam.net/) |  |  | [Package Draft](https://github.com/YunoHost-Apps/phpipam_ynh) |
| phplicensewatcher | A license manager |  | [Package Draft](https://github.com/YunoHost-Apps/phplicensewatcher_ynh) |
| [PHPList](https://www.phplist.com) |  | [Upstream](https://github.com/phpList) |  |
| [Phraseanet](https://docs.phraseanet.com/3.8/fr/index.html#) |  | [Upstream](https://github.com/alchemy-fr/Phraseanet-Docs) |  |
| [pia](https://github.com/LINCnil/pia) | A tool to help carrying out Privacy Impact Assessments |  | [Package Draft](https://github.com/YunoHost-Apps/pia_ynh) |
| Pico |  | [Upstream](https://github.com/picocms/Pico) |  |
| Playmaker |  | [Upstream](https://github.com/NoMore201/playmaker) |  |

> | [plonecms](plone.org) | Create a modern Website with a CMS written in python |  | [Package Draft](https://github.com/YunoHost-Apps/plonecms_ynh) |

Reviewer 2: ```suggestion
| [plonecms](plone.org) | Create a modern website with a CMS written in Python |  | [Package Draft](https://github.com/YunoHost-Apps/plonecms_ynh) |
```

| [Presentator](https://presentator.io/) | A design collaboration platform |  |  |

> | [prestashop](https://www.prestashop.com/) | Create a E-commerce Website |  | [Package Draft](https://github.com/YunoHost-Apps/prestashop_ynh) |

Reviewer 2: ```suggestion
| [prestashop](https://www.prestashop.com/) | Create an e-commerce website |  | [Package Draft](https://github.com/YunoHost-Apps/prestashop_ynh) |
```

| [Privoxy](https://www.privoxy.org) |  |  |  |
| [ProcessMaker](https://www.processmaker.com) |  |  |  |
| proftpd |  |  | [Package Draft](https://github.com/abeudin/proftpd_ynh) |
| ProtonMail’s WebClient |  | [Upstream](https://github.com/ProtonMail/WebClient) |  |

> | [psitransfer](https://psi.cx/tags/PsiTransfer/) | Transfer files or images without problem |  | [Package Draft](https://github.com/YunoHost-Apps/psitransfer_ynh) |

Reviewer 2: ```suggestion
| [psitransfer](https://psi.cx/tags/PsiTransfer/) | Transfer files or images without problems |  | [Package Draft](https://github.com/YunoHost-Apps/psitransfer_ynh) |
```

| [pterodactyl](https://pterodactyl.io/) |  |  | [Package Draft](https://github.com/YunoHost-Apps/pterodactyl_ynh) |
| [pufferpanel](https://emby.media/) |  |  | [Package Draft](https://github.com/YunoHost-Apps/pufferpanel_ynh) |
| Pump.io | File sharing and synchronization | [Upstream](https://github.com/pump-io/pump.io) |  |
| pydio | File sharing platform |  | [Package Draft](https://github.com/YunoHost-Apps/pydio_ynh) |
| [Pydio Cells](https://pydio.com/) |  | [Upstream](https://github.com/pydio/cells) |  |
| [pyload](https://pyload.net/) |  | [Upstream](https://github.com/pyload/pyload) | [Package Draft](https://github.com/YunoHost-Apps/pyload_ynh) |
| qBittorrent |  | [Upstream](https://github.com/qbittorrent/qBittorrent) |  |
| [Questions2answer](https://www.question2answer.org/) |  |  |  |
| [racktables](https://racktables.org) |  | [Upstream](https://github.com/RackTables/racktables) |  |
| Radarr |  | [Upstream](https://github.com/Radarr/Radarr) |  |
| [redmine](https://www.redmine.org/) | A flexible project management web application |  | [Package Draft](https://github.com/YunoHost-Apps/redmine_ynh) |

> | RemoteStorage | A remoteStorage server implementation written in PHP | [Upstream](https://github.com/fkooman/php-remote-storage) | [Package Draft](https://github.com/YunoHost-Apps/RemoteStorage_ynh) |

Reviewer 2: ```suggestion
| remoteStorage | A remoteStorage server implementation written in PHP | [Upstream](https://github.com/fkooman/php-remote-storage) | [Package Draft](https://github.com/YunoHost-Apps/RemoteStorage_ynh) |
```

| [Request Tracker](https://bestpractical.com) |  | [Upstream](https://github.com/bestpractical/rt) |  |
| [Restya](https://restya.com) |  | [Upstream](https://github.com/RestyaPlatform/board/) |  |
| [Retroshare](https://retroshare.cc/) |  | [Upstream](https://github.com/RetroShare/RetroShare) |  |

> | [roadiz](https://www.roadiz.io) | Create a modern Website |  | [Package Draft](https://github.com/YunoHost-Apps/roadiz_ynh) |

Reviewer 2: ```suggestion
| [roadiz](https://www.roadiz.io) | Create a modern website |  | [Package Draft](https://github.com/YunoHost-Apps/roadiz_ynh) |
```

> | [rocketchat](https://rocket.chat) | the ultimate chat platform |  | [Package Draft](https://github.com/YunoHost-Apps/rocketchat_ynh) |

Reviewer 2: ```suggestion
| [rocketchat](https://rocket.chat) | A chat platform |  | [Package Draft](https://github.com/YunoHost-Apps/rocketchat_ynh) |
```

| rs-short | An URL shortener | [Upstream](https://git.42l.fr/42l/rs-short) |  |

> | [rspamdui](https://rspamd.com/webui) | An web UI for rspamd spam filtering system |  | [Package Draft](https://github.com/YunoHost-Apps/rspamdui_ynh) |

Reviewer 2: ```suggestion
| [rspamdui](https://rspamd.com/webui) | A web UI for the Rspamd spam filtering system |  | [Package Draft](https://github.com/YunoHost-Apps/rspamdui_ynh) |
```
As per https://rspamd.com/

| rutorrent | Torrent client |  | [Package Draft](https://github.com/CotzaDev/rutorrent_ynh) |
| [sat](https://salut-a-toi.org) | An all-in-one tool to manage all your communications |  | [Package Draft](https://github.com/YunoHost-Apps/sat_ynh) |
| [ScenariChain-server](https://download.scenari.software/SCENARIchain-server/) |  |  |  |

> | [Schleuder](https://schleuder.org/schleuder/docs/concept.html) | A gpg-enabled mailing list manager |  |  |

Reviewer 2: ```suggestion
| [Schleuder](https://schleuder.org/schleuder/docs/concept.html) | A GPG-enabled mailing list manager |  |  |
```

| scm-manager | Share and manage Git, Mercurial and Subversion repositories |  | [Package Draft](https://github.com/drfred1981/scm-manager_ynh) |
| scrumblr | Software for notes |  | [Package Draft](https://github.com/YunoHost-Apps/scrumblr_ynh) |
| [Scuttlebutt Pub](https://www.scuttlebutt.nz/contributing) |  |  |  |
| [seenthis](https://www.seenthis.net/) | Short-blogging destiné à la veille d’actualité |  | [Package Draft](https://github.com/magikcypress/seenthis_ynh) |

> | [shadowsocks](https://shadowsocks.org) | A socks5 proxy to protect your Internet trafic |  | [Package Draft](https://github.com/YunoHost-Apps/shadowsocks_ynh) |

Reviewer 2: ```suggestion
| [shadowsocks](https://shadowsocks.org) | A SOCKS5 proxy to protect your Internet traffic |  | [Package Draft](https://github.com/YunoHost-Apps/shadowsocks_ynh) |
```

| [teampass](https://www.teampass.net) | Passwords Manager |  | [Package Draft](https://github.com/YunoHost-Apps/teampass_ynh) |
| [Technitium DNS](https://technitium.com/dns/) |  | [Upstream](https://github.com/TechnitiumSoftware/DnsServer) |  |
| [Teddy.io](https://teedy.io/) | Document manager | [Upstream](https://github.com/sismics/docs) |  |
| telegram_chatbot | Telegram chatbot - mini chatbot for Telegram |  | [Package Draft](https://github.com/YunoHost-Apps/telegram_chatbot_ynh) |
| TellForm |  | [Upstream](https://github.com/tellform/tellform) |  |
| [tes3mp](https://tes3mp.com/) |  |  | [Package Draft](https://github.com/YunoHost-Apps/tes3mp_ynh) |
| Thredded |  | [Upstream](https://github.com/thredded/thredded) |  |
| [TMate](https://tmate.io/) |  | [Upstream](https://github.com/tmate-io/tmate) |  |
| torrelay |  |  | [Package Draft](https://github.com/matlink/torrelay_ynh) |
| Tracim |  | [Upstream](https://github.com/tracim/tracim) |  |
| transpay | Interface to receive and manage donations with Stripe |  | [Package Draft](https://github.com/YunoHost-Apps/transpay_ynh) |

> | transwhat | A getway to whatsapp from Jabber |  | [Package Draft](https://github.com/Josue-T/transwhat_ynh) |

Reviewer 2: ```suggestion
| transwhat | A gateway to WhatsApp from Jabber |  | [Package Draft](https://github.com/Josue-T/transwhat_ynh) |
```
As per https://no.wikipedia.org/wiki/WhatsApp

| [Tryton](https://www.tryton.org/) | A solid ERP system |  |  |
| [turtl](https://turtlapp.com/) | A secure collaboratite notebook |  | [Package Draft](https://github.com/YunoHost-Apps/turtl_ynh) |

> | tutao | End-To-End Encrypted Mail Client | [Upstream](https://github.com/tutao/tutanota/) | [Package Draft](https://github.com/YunoHost-Apps/tutao_ynh) |

Reviewer 2: ```suggestion
| tutao | End-to-end encrypted e-mail client | [Upstream](https://github.com/tutao/tutanota/) | [Package Draft](https://github.com/YunoHost-Apps/tutao_ynh) |
```

| Twister |  | [Upstream](https://github.com/miguelfreitas/twister-core/) |  |

> | [TwitRSS.me](https://twitrss.me/) | Create RSS feeds from twitter | [Upstream](https://github.com/ciderpunx/twitrssme) |  |

Reviewer 2: ```suggestion
| [TwitRSS.me](https://twitrss.me/) | Create RSS feeds from Twitter | [Upstream](https://github.com/ciderpunx/twitrssme) |  |
```

| webogram | Webogram - a new era of messaging | [Upstream](https://github.com/zhukov/webogram) | [Package Draft](https://github.com/YunoHost-Apps/webogram_ynh) |
| [WebThings Gateway](https://iot.mozilla.org/gateway/) |  | [Upstream](https://github.com/mozilla-iot/) |  |
| [webtrees](https://www.webtrees.net) | Web-based genealogy |  | [Package Draft](https://github.com/YunoHost-Apps/webtrees_ynh) |
| Whoogle | A metasearch engine | [Upstream](https://github.com/benbusby/whoogle-search) |  |
| [wildfly](https://wildfly.org) |  |  | [Package Draft](https://github.com/YunoHost-Apps/wildfly_ynh) |
| [Wireguard](https://www.wireguard.com/) |  | [Upstream](https://git.zx2c4.com/WireGuard/) |  |
| Wisemapping | An online mind mapping editor | [Upstream](https://bitbucket.org/wisemapping/wisemapping-open-source) | [Package Draft](https://github.com/YunoHost-Apps/wisemapping_ynh) |
| Xibo | A FLOSS digital signage solution | [Upstream](https://github.com/xibosignage) |  |
| [Xonotic](https://xonotic.org) |  | [Upstream](https://gitlab.com/xonotic) |  |
| yacy | Free and decentrelized search engine |  | [Package Draft](https://github.com/YunoHost-Apps/yacy_ynh) |

> | [yellowcms](https://datenstrom.se/yellow/) | CMS for simple website |  | [Package Draft](https://github.com/YunoHost-Apps/yellowcms_ynh) |

Reviewer 2: ```suggestion
| [yellowcms](https://datenstrom.se/yellow/) | CMS for simple websites |  | [Package Draft](https://github.com/YunoHost-Apps/yellowcms_ynh) |
```

| [Yggdrasil](https://yggdrasil-network.github.io/) |  | [Upstream](https://github.com/yggdrasil-network/yggdrasil-go) |  |
| youtube-dl-webui | Web interface for youtube-dl |  | [Package Draft](https://github.com/YunoHost-Apps/youtube-dl-webui_ynh) |
| yunofav | A page of favorite links |  | [Package Draft](https://github.com/YunoHost-Apps/yunofav_ynh) |

> | [yunohost](https://yunohost.org) | Yunohost in Yunohost, crazy :D ! |  | [Package Draft](https://github.com/aymhce/yunohost_ynh) |

Reviewer 2: ```suggestion
| [yunohost](https://yunohost.org) | YunoHost in YunoHost, crazy :D ! |  | [Package Draft](https://github.com/aymhce/yunohost_ynh) |
```
